collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -21,9 +21,6 @@ TIMEOUT=60
 PRIORITY_NICE=19 # lowest
 PRIORITY_IONICE="idle" # lowest
 
-# Set the dmesg date to use English, as non-English environments such as Japanese will display non-English characters in it.
-LANG=C
-
```

Comment:
Not sure why this is being removed?

Reply:
strange how this ended up in this PR this was not supposed to be modified either